### ISSUE TITLE
Transaction not broadcasting

### ISSUE THREAD

**Issue by adityanithariya** [bitcoinjs/bitcoinjs-lib#2027]
But when checking the history of the account, which is used to do the transfer, it is not having any such transaction listed there

_Originally posted by @adityanithariya in https://github.com/bitcoinjs/bitcoinjs-lib/issues/2026#issuecomment-1878603360_

**Comment by adityanithariya** [comment]
@junderw Please provide a solid source by which I can learn about creating bitcoin transaction. from fetching UTXOs to broadcasting transactions, as examples are not having this thing there

**Comment by junderw** [comment]
Learn more about Bitcoin here: https://btcinformation.org/en/developer-guide

`dust` means that one of the outputs has a value that is too small.

Usually you want to make sure all outputs in your transaction are 546 satoshis or larger.

Stop creating new issues. These are all related to the same issue.

If you want the issue to stay open, then make the issue's topic more general. You ask "why is my transaction not XYZ?" and I answered it so I closed it. If you have follow up questions, then do them in the same issue.

Since I explained the `dust` error. Your "Transaction not broadcasting" issue is now closed.
